bin/n

```diff
@@ -740,8 +740,7 @@ activate() {
       printf '\nNote: the node command changed location and the old location may be remembered in your current shell.\n'
       log old "${original_node}"
       log new "${active_node}"
-      # shellcheck disable=SC2016
-      printf 'To reset the command location hash either start a new shell, or execute PATH="$PATH"\n'
+      printf 'If "node --version" shows the old version then start a new shell, or reset the location hash with:\nhash -r  (for bash, zsh, ash, dash, and ksh)\nrehash   (for csh and tcsh)'
```

Comment:
There needs to be a trailing `/n` included in the string, as printf does not add a newline.

Reply:
Nice catch! See f7471e6.